{"filename": "haskell-enclosed-exceptions-1.0.3-320-x86_64.pkg.tar.zst", "name": "haskell-enclosed-exceptions", "base": "haskell-enclosed-exceptions", "version": "1.0.3-320", "desc": "Catching all exceptions from within an enclosed computation", "csize": "13508", "isize": "40187", "md5sum": "d9a7dc099cd0d118b943e6ea002f88f8", "sha256sum": "d5ebcc9848ae74b20f666aa5369b4097ac8c7a89fbaf194083ede3a4f9598b80", "pgpsig": "iQIzBAABCgAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmi6OeEACgkQeGxj8zDXy5IIAA//e9MkLIUk+SEWQIjAzpuYYdTPs8y8FJr2GvDj52RNwgAWCqhQE8TBa5ORDot9GlOX5OdXHrsWZr4q0j47UzKCytN0ZwDT2iM+eUp9nUoQ85huEXfsuIXiudE0diKR0kaN6Jh76XnUKeD3HTZmNwbFMwRpZfYtCW1969g+wlWcylRTzFo680ohoa5KsQp9sSCuHz/rexXFD5qa3aj6TaXnsHp9gTaq8KRA7PWJWa2yYZlndzcx0IvJanT7spCRvCIQ+84yXIOn2g9nqceoF3QnCaLgVf0K4VkDOPf2tT4mG5vIt0hSzhq2VnD5aiG1rCYtGMhQICHYvI/TL7shKgxIm0qO2pnNTrshujgL9oC4TXCF12oAT+nUThWSxZqiEqUIG7ppwhoK40Y3yjhw8gJDeInguhzTtM/5ckhEixXBtT1GP9r5fPRyyY0f22bz+PA74lQFEDCsou4XycObGG2tzyIyZdXc45RwwueszU2KzSOiNZ/2hYCMbdOb/uovsAMg4pnWj2I4k2upbcD6OOHm+MTLVtndqkvtImisLeWEq/XKElEgfSmuF+rP+6EnBtuNOtgCNqYGYx3HQ4j/l5BtclATweXcoXGyoDEElQSceaUj0E/YSqr4DChJhsBtK9n4cNOvo8wktufS1DvtUG5WItay75PntKpuHQ8Jo0dl+RI=", "url": "https://github.com/jcristovao/enclosed-exceptions", "license": "MIT", "arch": "x86_64", "builddate": "1757034955", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["ghc-libs", "haskell-lifted-base", "haskell-monad-control", "haskell-transformers-base"], "makedepends": ["ghc"]}